src/utils/preparedTodos.ts

```diff
@@ -0,0 +1,35 @@
+import { TodoStatus } from '../types/TodoStatus';
+import { Todo } from '../types/Todo';
+
+export function preparedTodos(
```

Comment:
function name should contain some action
`preparedTodos` is good name for the variable, which will store the final array
let's think more about function naming